<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ambilight</class>
 <widget class="QWidget" name="Ambilight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>85</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="ScreenRegion" name="screenRegion" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>421</width>
     <height>241</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string extracomment="choose region to analyse"/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="colorPallete" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>191</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScreenRegion</class>
   <extends>QWidget</extends>
   <header location="global">screenregion.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
